components/automate-chef-io/content/docs/quickstart.md

```diff
@@ -14,46 +14,88 @@ toc = true
 Hello and welcome to Chef Automate 2! This document guides you through the initial installation and trial activation.
 
 The easiest way to start exploring Chef Automate is to install it on a local VM.
-We recommend using [Vagrant][] to create your test environment.
+We recommend using [Vagrant](https://www.vagrantup.com/) to create your test environment.
 Use this Vagrantfile to provision a VM. Chef recommends a minimum of 4GB RAM.
```

Comment:
"this Vagrantfile" is currently pointing to something that doesn't exist yet. I'd probably change the opening to something like 

```
Hello and welcome to Chef Automate 2! This document guides you through setting up a Vagrant VM test environment on which to install Automate and activate a trial. It covers setting up the Vagrant environment and creating an example Vagrantfile.
```

Reply:
`use this Vagrantfile`, followed by the Vagrantfile code works.